### PR TITLE
Remove array editing from game pack

### PR DESCRIPTION
**Pull request by Banane9**
Extracted to Mod in https://github.com/ResoniteModdingGroup/ArrayEditing

### CODE REVIEW

**Comment by Nytra** [comment]
This does work but is missing patches for the other SyncMemberEditorBuilder methods and their default handlers

**Comment by Banane9** [comment]
Yeee, mostly created the PR as a reminder yesterday and to see what changed 😅

File: MonkeyLoader.Resonite.Integration/Locale/en.json [hunk 1]
```diff
@@ -75,6 +75,12 @@
     "MonkeyLoader.GamePacks.Resonite.ModLocaleFallbackDriver.Description": "Ensures that mods' locale messages show up as the (English) fallback value for users without the mod adding them.",
     "MonkeyLoader.GamePacks.Resonite.ModSettingStandaloneFacet.Description": "Allows creating working standalone facets from MonkeyLoader Mod Settings items.",
     "MonkeyLoader.GamePacks.Resonite.SettingsDataFeedInjector.Description": "Adds the MonkeyLoader category to the Settings.",
+    "MonkeyLoader.GamePacks.Resonite.SyncMemberEditorBuilderInjector.Description": "Sends out the various Build Member Editor events to allow monkeys to override the generation them.",
```
Comment (Nytra) [inline MonkeyLoader.Resonite.Integration/Locale/en.json:78]: typo

File: MonkeyLoader.Resonite.Integration/UI/Inspectors/DefaultBuildMemberEditorHandlers.cs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    internal sealed class DefaultBuildMemberEditorHandlers : ResoniteMonkey<DefaultBuildMemberEditorHandlers>,
+        ICancelableEventHandler<BuildArrayEditorEvent>, ICancelableEventHandler<BuildBagEditorEvent>,
+        ICancelableEventHandler<BuildListEditorEvent>, ICancelableEventHandler<BuildPlaybackEditorEvent>,
+        ICancelableEventHandler<BuildFieldEditorEvent>, ICancelableEventHandler<BuildObjectEditorEvent>
+    {
+        public int Priority => HarmonyLib.Priority.Normal;
+
+        public bool SkipCanceled => true;
+
+        void ICancelableEventHandler<BuildArrayEditorEvent>.Handle(BuildArrayEditorEvent eventData) => throw new NotImplementedException();
```
Comment (Nytra) [inline MonkeyLoader.Resonite.Integration/UI/Inspectors/DefaultBuildMemberEditorHandlers.cs:27]: Why does this throw?

Comment (Banane9) [reply MonkeyLoader.Resonite.Integration/UI/Inspectors/DefaultBuildMemberEditorHandlers.cs:27]: Oh, woops, that should have been implemented like the others lol

**Pull request merged**